src/libs/protocols/hack/update-generated-proto.sh

```diff
@@ -47,19 +47,20 @@ show_usage() {
 }
 
 generate_go_sources() {
-    local cmd="protoc -I$GOPATH/src:$GOPATH/src/github.com/kata-containers/kata-containers/src/libs/protocols/protos \
+    local cmd="protoc -I$GOPATH/src:$(realpath ../)/libs/protocols/protos \
 --gogottrpc_out=plugins=ttrpc+fieldpath,\
 import_path=github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols/grpc,\
-\
-Mgithub.com/kata-containers/kata-containers/src/libs/protocols/protos/csi.proto=github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols/grpc,\
-\
-Mgithub.com/kata-containers/kata-containers/src/libs/protocols/protos/types.proto=github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols,\
-\
-Mgithub.com/kata-containers/kata-containers/src/libs/protocols/protos/oci.proto=github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols/grpc,\
-\
```

Comment:
why is this being removed?

Reply:
I tried to explain at the commit message:

>   The definitions are already specified in the .proto files using the
>     go_package option. Centralizing them in one location reduces the
>     potential for errors and simplifies the script.

Any reason we should keep it duplicated?

Reply:
No, i didn't catch that